test/fixtures/dialects/oracle/prompt-alter.yml

```diff
@@ -0,0 +1,124 @@
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> The lifetime\
+        \ budget amount to spend during the lifetime of the campaign group, which\
+        \ is derived from the runSchedule, used only if the campaign group is enabled\
+        \ for dynamic budget optimization'"
+- statement_terminator: ;
+- statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: CAMPAIGN_GROUPS
+      - dot: .
+      - identifier: WEEKLY_BUDGET
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> The weekly delivery\
+        \ goal and charging limit of a campaign group, used only if the campaign group\
+        \ is enabled for dynamic budget optimization'"
+- statement_terminator: ;
+- statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: CAMPAIGN_GROUPS
+      - dot: .
+      - identifier: WEEKLY_BUDGET_UPDATE_TIME
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> Indicates the\
+        \ last timestamp the weekly budget was updated, used only if the campaign\
+        \ group is enabled for dynamic budget optimization'"
+- statement_terminator: ;
+- statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: CAMPAIGN_GROUPS
+      - dot: .
+      - identifier: OPTIMIZATION_TARGET
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> Indicates how\
+        \ this campaign group is optimized, used only if the campaign group is enabled\
+        \ for dynamic budget optimization'"
+- statement_terminator: ;
+- statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: CAMPAIGN_GROUPS
+      - dot: .
+      - identifier: COST_TYPE
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> Indicates whether\
+        \ this campaign group is charged per impression, per click, or other model,\
+        \ used only if the campaign group is enabled for dynamic budget optimization'"
+- statement_terminator: ;
+- statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: CAMPAIGN_GROUPS
+      - dot: .
+      - identifier: DYNAMIC_BUDGET_OPT_ENABLED
+    - keyword: IS
+    - literal: "'<compliance> {\"dataType\": \"NONE\"} </compliance> Indicates whether\
+        \ this campaign group is enabled for Dynamic Budget Optimization, which dynamically\
+        \ allocates budget across campaigns'"
+- statement_terminator: ;
+- statement:
+    unparsable:
+    - raw: SHOW
+    - raw: ERRORS
+- statement_terminator: ;
```

Comment:
Here's where you see it failing. Want to check there are no `unparsable` section in the generated YML fails or the test suite won't complete.

```suggestion
```

Reply:
Sweet, removed this file for now until I come back to prompt 💯